lib/tasks/import.rake

```diff
@@ -79,13 +79,12 @@ namespace :csv_load do
     end
   end
 
-  task :all do
-    ENV['ENVIROMENT'] = 'true'
+  task all: :environment do
     Rake::Task["csv_load:customers"].execute
-    Rake::Task["csv_load:invoice_items"].execute
-    Rake::Task["csv_load:items"].execute
     Rake::Task["csv_load:merchants"].execute
-    Rake::Task["csv_load:transactions"].execute
+    Rake::Task["csv_load:items"].execute
     Rake::Task["csv_load:invoices"].execute
+    Rake::Task["csv_load:transactions"].execute
+    Rake::Task["csv_load:invoice_items"].execute
```

Comment:
Do these rakes need to be in a specific order?

Reply:
Yeah, if you run the belongs_to before the has_many they fizzle because they can't find the foreign key they need.

Reply:
Gotcha, makes sense